proposed/http-client/http-client.md

```diff
@@ -30,15 +30,15 @@ should not follow redirect nor throw exceptions on HTTP responses with status 4x
 The following interfaces MAY be implemented together within a single class or
 in separate classes.
 
-### HttpClientInterface
+### ClientInterface
 
 ```php
 namespace Psr\Http\Client;
 
 use Psr\Http\Message\RequestInterface;
 use Psr\Http\Message\ResponseInterface;
 
-interface HttpClient
+interface Client
```

Comment:
is PSR standard naming with Interface suffix or without? i think we said we do without suffix, but then the title of this section should be `### Client interface` i think.

Reply:
Thank you

Reply:
Ah, I was hoping noone notices :trollface:

Reply:
Will need to be `ClientInterface` yep

Reply:
Thank you